Taylor/UI/UIImage.swift

```diff
@@ -74,4 +74,22 @@ extension UIImage
         
         return tintedImage
     }
+
+    /**
+     Creates a new instance of an image resized with the specified ratio
+
+     - parameter ratio: The desired ratio
+     - returns: New instance of UIImage
+     */
+    func resize(ratio: CGFloat) -> UIImage? {
+        return resizeImage(to: size.applying(CGAffineTransform(scaleX: ratio, y: ratio)))
+    }
+
+    private func resizeImage(to size: CGSize) -> UIImage? {
+        UIGraphicsBeginImageContextWithOptions(size, false, scale)
```

Comment:
I would put the param `opaque` as a param with a default value value of `false` to make it possible to save the image without a potentially useless alpha channel, witch takes extra disk space.